platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationImpl.java

```diff
@@ -124,7 +124,7 @@ private void setRepoNameAndVersion(String repoName) {
             name = repoNameParts[0];
             version = repoNameParts[1];
             break;
-        case 3:
+        default:
             throw new RuntimeException(String.format("Could not tokenize repository name of '%s'",
```

Comment:
I feel that this is a little too rigid now. A malformed (according to us) repo name shouldn't throw a `RuntimeException`. I would rather log a warning and use some sensible defaults instead.